src/renderer/components/Lofi/Cover/Controls/index.tsx

```diff
@@ -39,31 +39,58 @@ class Controls extends React.Component<any, any> {
     this.props.parent.togglePlayPause();
   }
 
-  async forward() {
+  async forward(e: React.MouseEvent) {
     if (this.props.parent.state.spotifyError) {
       return;
     }
 
-    SpotifyApiInstance.fetch('/me/player/next', {
-      method: 'POST',
-    }).then(() => {
-      // Spotify API doesn't update fast enough sometimes, so give it some leeway
-      setTimeout(this.props.parent.listeningTo.bind(this), 2000);
-    });
+    if (e.ctrlKey) {
+      SpotifyApiInstance.fetch(`/me/player`, {
+        method: 'GET',
+      }).then((body) => {
+        const currentProgress = Number(body.progress_ms);
+        SpotifyApiInstance.fetch(`/me/player/seek?position_ms=${currentProgress + 15000}`, {
```

Comment:
FYI I'd rather have this `15000ms` as a configurable setting, but for now it can stay that way. I'm in the process of refactoring the whole application, I'll include your change and expose the settings in the settings window...

Reply:
Yeah, I also thought about doing that, but I wanted to start with smaller changes.